### DISCUSSION TITLE
Android Imported Project Location (add Android > media)

### DISCUSSION THREAD

**Discussion by dewey-b** [opengisch/QField#4526] [Ideas]
Currently, the QField imported projects directory is located at Android  > data > ch.opengis.qfield > files > Imported Projects

This location works fine if I want to use a USB cable to transfer data back and forth between my PC and my phone. It would be more convenient for me if I could use "nearby share" on my PC and Android phone to transfer data back and forth. However, "nearby share" does not provide access to the Android > __data__ directory. If QField could also look in the Android > __media__ directory (perhaps Android  > media > ch.opengis.qfield > files > Imported Projects), it would allow for easy, wireless transfer of data to and from my PC to my phone.

Thank you for your consideration.

**Comment by Dartazen** [comment]
Hi, I would like to bring up this suggestion because since recent system updates, access to the Android/data folder has become impossible on the phone. It would be really nice to be able to configure the folder in which the written files are stored (Android/media would be great, as is the case with other apps)!

**Comment by Spacedatists** [comment]
+1 this is a must, since latest Andoid updates totally screwed up our sync-system. No down-syncs to /data-folder, nor upload-syncs from individual DCIM project folders.

**Comment by Spacedatists** [comment]
... so I made a QFIELD feature [request](https://ideas.qfield.org/app-feature-requests/p/change-directory-of-imported-projects-folder-on-android-devices).
please vote for it!

**Comment by AveryAndrews** [comment]
Or just give QField a dialog specifying any generally editable folder (on computer via USB cables, file managers, autosync or similar) on the device for the project.  Re-importing projects into android/data/ is too clunky to be viable, due to the necessity of deleting the original imported project, an easy step to forget, and then you don't know which version to keep, so have to delete both of them and import again.
